config/ui-extensions/apirules/form

```diff
@@ -151,7 +151,7 @@
     path: '/.*'
     methods: ['GET']
     accessStrategies:
-      - handler: 'allow'
+      - handler: 'allow_methods'
```

Comment:
This is the template configuration, that specifies the default structure. Since we want to propagate the usage of the new handler, we should have it as the new default.